packages/fxa-settings/src/components/App/index.test.tsx

```diff
@@ -101,3 +118,165 @@ describe('performance metrics', () => {
     expect(NavTiming.observeNavigationTiming).not.toHaveBeenCalled();
   });
 });
+
+describe('App component', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  it('renders `LoadingSpinner` component when loading initial state is true', async () => {
```

Comment:
Small nit: I don't think all of these need to be `async` functions, just the ones that `await` navigating